runelite-client/src/main/java/net/runelite/client/ui/ClientUI.java

```diff
@@ -770,6 +779,29 @@ private void toggleSidebar()
 		}
 	}
 
+	private void togglePluginPanel()
+	{
+		// Toggle sidebar open
+		boolean isPanelOpen = sidebarOpen;
+		sidebarOpen = !sidebarOpen;
+
+		if (isPanelOpen)
+		{
+			contract();
+		}
+		else
+		{
+			// Try to restore last panel
+			expand(currentNavButton);;
+
+			if (!container.isAncestorOf(pluginToolbar))
```

Comment:
what is this for?

Reply:
Without this, if you use the new hotkey to open the plugin panel, after closing the whole sidebar with the old hotkey, you get the plugin panel *without* the actual narrow sidebar.